packages/Uploader/src/components/File/File.styles.js

```diff
@@ -81,22 +68,14 @@ export const ProgressBarWrapper = styled.div`
   overflow: hidden;
 `;
 
-export const ProgressBar = styled.div`
-  background: ${tokens.color.purple};
-  height: 100%;
-  width: ${props => `${props.progress}%`};
+export const ProgressBar = styled.div(
+  ({ status, progress }) => css`
+    background: ${tokens.color.purple};
+    height: 100%;
+    width: ${`${progress}%`};
```

Comment:
wouldn't `${progress}%; `should work? the percentage is a valid string under css